src/EA.Weee.Web/Areas/AatfReturn/Mappings/ToViewModel/AddressTonnageSummaryToReusedOffSiteSummaryListViewModelMap.cs

```diff
@@ -24,10 +24,14 @@ public class AddressTonnageSummaryToReusedOffSiteSummaryListViewModelMap : IMap<
         public ReusedOffSiteSummaryListViewModel Map(AddressTonnageSummary source)
         {
             Guard.ArgumentNotNull(() => source, source);
-             
+
             foreach (var address in source.AddressData)
             {
-                AddressDataSummaries.Add(address);
+                var addressDataSummary = new AddressDataSummary();
```

Comment:
This this was the original design for the map but was changed by Will last week, was there an issue with it that we're changing this back?

Reply:
The original design was correct.  Will followed the mockup, but the mockup is out of date and he wasn't told